plugins/BEdita/Core/config/Migrations/20240930121402_TreesAddParams.php

```diff
@@ -0,0 +1,26 @@
+<?php
+declare(strict_types=1);
+
+use Migrations\AbstractMigration;
+
+class TreesAddParams extends AbstractMigration
+{
+    /**
+     * Change Method.
+     *
+     * @return void
+     */
+    public function change()
+    {
+        $columnTypes = $this->getAdapter()->getColumnTypes();
+        $type = in_array('json', $columnTypes) ? 'json' : 'text';
+        $this->table('trees')
+            ->addColumn('params', $type, [
+                'comment' => 'Folder parameters (JSON)',
```

Comment:
Maybe a pedantry, but these are not "folder parameters", but rather parameters associated with the tree position of an object.

```suggestion
                'comment' => 'Parameters for the position on tree (JSON)',
```